src/common/arrow/arrow_row_batch.cpp

```diff
@@ -70,12 +70,33 @@ void ArrowRowBatch::templateInitializeVector<LogicalTypeID::ARRAY>(ArrowVector*
     vector->childData.push_back(std::move(childVector));
 }
 
+template<>
+void ArrowRowBatch::templateInitializeVector<LogicalTypeID::MAP>(ArrowVector* vector,
+    const LogicalType& type, std::int64_t capacity) {
+    return templateInitializeVector<LogicalTypeID::LIST>(vector, type, capacity);
+}
+
 template<>
 void ArrowRowBatch::templateInitializeVector<LogicalTypeID::STRUCT>(ArrowVector* vector,
     const LogicalType& type, std::int64_t capacity) {
     initializeStructVector(vector, type, capacity);
 }
 
+template<>
+void ArrowRowBatch::templateInitializeVector<LogicalTypeID::UNION>(ArrowVector* vector,
+    const LogicalType& type, std::int64_t capacity) {
+    // Interestingly, unions don't have their own validity bitmap
```

Comment:
Are u aware why is that? maybe put a link to arrow's doc here? so the comment is more self-explanatory.

Reply:
Added link to doc :+1: